Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -284,6 +284,9 @@ static wxString true_color_desc =
     wxTRANSLATE("Forces the game to render the RGB color channels in 24-bit, thereby increasing "
                 "quality by reducing color banding.\nIt has no impact on performance and causes "
                 "few graphical issues.\n\n\nIf unsure, leave this checked.");
+static wxString vertex_rounding_desc =
+    wxTRANSLATE("Round vertices to whole pixels.  Fixes some "
```

Comment:
Could we make this "Round 2D vertices to whole pixels"

I feel like this better explains what it does.